src/AmitaR/Q1LargestOfThreeNumbers.java

```diff
@@ -0,0 +1,46 @@
+//1. Find the largest of three numbers using Logical And operator.
+
+package AmitaR;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+public class Q1LargestOfThreeNumbers
+{
+	void findLarge(int num[])
+	{
```

Comment:
this condition failed in case of when someone pass array
{7, 4, 5}
first biggest, smallest num, second biggest